2021-06-14 09:27:17,498 INFO    MainThread:12242 [wandb_setup.py:_flush():69] setting env: {}
2021-06-14 09:27:17,498 INFO    MainThread:12242 [wandb_setup.py:_flush():69] setting login settings: {}
2021-06-14 09:27:17,499 INFO    MainThread:12242 [wandb_init.py:_log_setup():337] Logging user logs to /home/treoroygbiv/Projects/null_it_out_implementation/null-it/wandb/run-20210614_092717-mc4a006y/logs/debug.log
2021-06-14 09:27:17,499 INFO    MainThread:12242 [wandb_init.py:_log_setup():338] Logging internal logs to /home/treoroygbiv/Projects/null_it_out_implementation/null-it/wandb/run-20210614_092717-mc4a006y/logs/debug-internal.log
2021-06-14 09:27:17,499 INFO    MainThread:12242 [wandb_init.py:init():370] calling init triggers
2021-06-14 09:27:17,499 INFO    MainThread:12242 [wandb_init.py:init():375] wandb.init called with sweep_config: {}
config: {}
2021-06-14 09:27:17,499 INFO    MainThread:12242 [wandb_init.py:init():419] starting backend
2021-06-14 09:27:17,499 INFO    MainThread:12242 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-06-14 09:27:17,518 INFO    MainThread:12242 [backend.py:ensure_launched():135] starting backend process...
2021-06-14 09:27:17,534 INFO    MainThread:12242 [backend.py:ensure_launched():139] started backend process with pid: 12266
2021-06-14 09:27:17,535 INFO    MainThread:12242 [wandb_init.py:init():424] backend started and connected
2021-06-14 09:27:17,536 INFO    MainThread:12242 [wandb_init.py:init():472] updated telemetry
2021-06-14 09:27:17,537 INFO    MainThread:12242 [wandb_init.py:init():491] communicating current version
2021-06-14 09:27:18,334 INFO    MainThread:12242 [wandb_init.py:init():496] got version response 
2021-06-14 09:27:18,334 INFO    MainThread:12242 [wandb_init.py:init():504] communicating run to backend with 30 second timeout
2021-06-14 09:27:19,230 INFO    MainThread:12242 [wandb_init.py:init():529] starting run threads in backend
2021-06-14 09:27:21,297 INFO    MainThread:12242 [wandb_run.py:_console_start():1624] atexit reg
2021-06-14 09:27:21,297 INFO    MainThread:12242 [wandb_run.py:_redirect():1498] redirect: SettingsConsole.REDIRECT
2021-06-14 09:27:21,298 INFO    MainThread:12242 [wandb_run.py:_redirect():1503] Redirecting console.
2021-06-14 09:27:21,299 INFO    MainThread:12242 [wandb_run.py:_redirect():1559] Redirects installed.
2021-06-14 09:27:21,299 INFO    MainThread:12242 [wandb_init.py:init():554] run started, returning control to user process
2021-06-14 09:27:21,300 INFO    MainThread:12242 [wandb_run.py:_config_callback():873] config_cb None None {'dims': 64}
2021-06-14 09:27:21,484 INFO    MainThread:12242 [wandb_run.py:_atexit_cleanup():1594] got exitcode: 1
2021-06-14 09:27:21,484 INFO    MainThread:12242 [wandb_run.py:_restore():1566] restore
2021-06-14 09:27:23,867 INFO    MainThread:12242 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 784
  total_bytes: 784
}

